client/my-sites/stats/stats-detail-months/index.jsx

```diff
@@ -130,13 +130,10 @@ export default React.createClass( {
 				</div>
 				<div className="module-content">
 					<div className="module-content-text module-content-text-info">
```

Comment:
While we are in here, it might be nice to start dropping in some of the new components.  This could use the [StatsModuleContent](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/stats/stats-module/content-text.jsx)

`<StatsModuleContnet className="module-content-text-info">...</StatsModuleCotnent>`